src/main/resources/lang/en_US.json

```diff
@@ -212,7 +211,7 @@
     "guiScaleY": "GUI Y Scale: %scale%",
     "abbreviateSkillXpDenominator": "Abbreviate Skill XP Denominator",
     "otherDefenseStats": "Other Defense Stats (Soul Tether/Cell Alignment)",
-    "hideSpawnPointPlayers": "Hide Spawn Point Players",
+    "hideSpawnPointPlayers": "Hide Players at Hub Spawn Point",
```

Comment:
Does this not work in other islands that aren't the hub?

Reply:
No it doesn't, though that's a good idea to add in a future update.